### ISSUE TITLE
Fc not building due to missing crate

### ISSUE THREAD

**Issue by jw3** [ctc-oss/fapolicy-analyzer#929]
```
error: failed to select a version for the requirement `directories = "^4.0"`
candidate versions found which didn't match: 5.0.1, 3.0.2
```

https://github.com/ctc-oss/fapolicy-analyzer/actions/runs/6574210862/job/17859188326#step:8:1946

**Comment by jw3** [comment]
Seems to exist https://mirrors.kernel.org/fedora/releases/38/Everything/source/tree/Packages/r/rust-directories-4.0.1-3.fc38.src.rpm

**Comment by jw3** [comment]
Fc 37 has same issue

**Comment by jw3** [comment]
https://lists.fedoraproject.org/archives/list/devel-announce@lists.fedoraproject.org/thread/C6U5YUK5RPPWIWABKDVJYBSFOJ7RSOSN/

Delayed to 10-31-2023

**Comment by jw3** [comment]
0.5.0 will soon be on all platforms, going to bump crate to 0.5

https://src.fedoraproject.org/rpms/rust-directories

**Comment by jw3** [comment]
Closed in #945

**Comment by jw3** [comment]
Fedora 37 will go EOL on 2023-12-05
